addon/serializer.js

```diff
@@ -82,7 +82,7 @@ class Serializer {
    * @public
    */
   keyForCollection(modelName) {
-    return pluralize(this.keyForModel(modelName));
+    return toDbCollectionName(modelName);
```

Comment:
this helper should only be used when accessing a schema or db collection; I'd rather keep this the basic logic so people can see "oh, it's just munging the key used in the payload. I can do whatever I want here"